data/tutorials/getting-started/02_fst_program.md

```diff
@@ -0,0 +1,308 @@
+---
+id: how-to-write-an-ocaml-program
+title: How to Write an OCaml Program
+description: >
+  Get something real out of the damm French thing!
+category: "getting-started"
+---
+
+[TOC]
+
+## Get Something Done
+
+This tutorial is the last part of the three-part series. Please ensure you have completed [“Installing OCaml”](linkhere) and [`“A Tour of OCaml”](linkhere) before proceeding onto your first OCaml project. You will need your environment been set up as described in the ["Installing OCaml"](Install_link).
+
+In this tutorial, we start working with files containing OCaml source code and compiling them to produce excutable binaries. However, this is not a detailed tutorial on OCaml compilation, project modularisation, or dependencies management; it only gives a glimpse at those topics. The goal is to sketch the bigger picture before extensively presenting topics in order to avoid getting lost in the details. 
+
+In other words, breadth-first learning instead of depth-first learning.
+
+## Compiling OCaml Programs
+
+OCaml comes with both an interpreter and a compiler. In the previous tutorial, only the interpreter was used. This section gives a glimpse of how to use the compiler. 
+
+We start by seting up a traditional “Hello World!” project using Dune, OCaml’s build system. The following creates it under the name `hello`:
+
+```shell
+$ dune init proj hello
+Entering directory '/home/cuihtlauac/caml/ocaml.org/hello'
+Success: initialized project component named hello
+```
+
+The project is stored in a directory named `hello`. The `tree` command lists the files and directories created. 
+
+```shell
+$ cd hello
+$ tree
```

Comment:
I wasn't able to get this tree command to work. I was able to see the top directories with `ls`, but I got an error with `tree`. Did I do something wrong?

Reply:
Even with the updated settings from Friday, `tree` still doesn't work for me. Same error.

Reply:
Perhaps add "You might need to install `tree` to get this output. On Homebrow, the command is `brew install tree`.